modules/token-overview.adoc

```diff
@@ -0,0 +1,66 @@
+:_content-type: CONCEPT
+[id="token-overview"]
+= Introduction to {productname} OAuth 2.0 tokens
+
+The {productname} OAuth2 token system provides a secure, standards-based method for accessing {productname}'s API and other relevant resources. The {productname} OAuth2 token-based approaches provides a secure method for handling authentication and authorization for complex environments. Compared to more traditional API tokens, {productname}'s OAuth2 token system offers the following enhancements:
+
+* Standards-based security, which adheres to the link:https://oauth.net/2/[OAuth 2.0 protocol].
+* Revocable access by way of deleting the application in which the OAuth2 token exists. 
+* Fine-grained access control, which allows {productname} administrators the ability to assign specific permissions to tokens.
+* Delegated access, which allows third-party applications and services to act on behalf of a user.
+* Future-proofing, which helps ensure that {productname} remains compatible with other services, platforms, and integrations.
+
+The following sections provide more tails about the token types available with {productname}.
+
+[id="token-types"]
+== {productname} token types
+
+{productname} primarily supports two types of tokens: OAuth2 access tokens and robot account tokens. A third token type, an _OCI referrers access token_, that is required to list OCI referrers of a manifest under a repository, is also available when warranted.
+
+[discrete]
+[id="oauth2-access-tokens"]
+=== OAuth2 access tokens 
+
+OAuth 2 access tokens enable user-authenticated access to the {productname} API, suitable for applications that require user identity verification. These tokens are obtained through an OAuth2 authorization process, where a {productname} administrator generates a token on behalf of themselves or another user to access {productname} resources. These tokens provide scope-based access and specify what resources and actions the token holder is allowed. Currently, the following options can be selected for a token holder:
+
+* *Administer Organization*. When selected, allows the user to be able to administer organizations, including creating robots, creating teams, adjusting team membership, and changing billing settings.
+
+* *Administer Repositories*. When selected, provides the user administrator access to all repositories to which the granting user or robot account has access.
```

Comment:
Tokens cannot be issued on behalf of robot accounts. Only real users.

Reply:
Weird. That's the description from the Quay UI.

Reply:
I raised a PR to remove these references: https://github.com/quay/quay/pull/3426